Models/models.js

```diff
@@ -36,4 +36,21 @@ exports.fetchAllArticles = () => {
       return rows
     })
 
-}
\ No newline at end of file
+}
+
+exports.fetchArticleCommentsById = (article_id) => {
+    const query = `SELECT comments.comment_id, 
+    comments.votes, comments.created_at, comments.author, comments.body, comments.article_id
+    FROM comments
+    LEFT JOIN articles ON articles.article_id= comments.article_id
+    WHERE comments.article_id = $1
+    ORDER BY comments.created_at DESC;`
+    
+    return db.query(query, [article_id]).then(({rows})=>{
+        if(rows.length === 0) {
+            return Promise.reject({status: 404, msg: 'article_id does not exist'})
+          }
```

Comment:
You will need to add something to your model to handle valid articles that have no comments 🙂